src/store.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers';
+export default function configureStore() {
+ return createStore(
+    rootReducer,
+    compose(
+      applyMiddleware(thunk),
+      window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
This extension will make our test fail. I suggest removing it

Reply:
@AJAkimana he can use it, I will put a test environment on testing

Reply:
@AJAkimana we have used this extension with @Kabalisa when testing and it has no effect